tests/conftest.py

```diff
@@ -0,0 +1,64 @@
+import functools
+import logging
+import os
+import os.path
+
+import pytest
+
+from pytest_bdd import given
+from pytest_bdd import parsers
+from pytest_bdd import then
+from pytest_bdd import when
+
+from utils.helper import run_ansible_playbook
+
+
+
+def pytest_addoption(parser):
+    parser.addoption(
+        '--inventory', help='Inventory where is specified servers for tests'
+    )
+
+
+@pytest.fixture
+def pytestbdd_strict_gherkin():
+   return False
+
+
+@pytest.fixture(scope="session")
+def inventory_sources(request):
+    inventory_cli_arg = request.config.getoption('--inventory')
+    inventory_env_var = os.environ.get('ANSIBLE_INVENTORY')
+    inventory = inventory_cli_arg or inventory_env_var
+    assert inventory, 'You should provide an inventory to run test'
+    return inventory_cli_arg, inventory_env_var
+
+
+@given('A complete inventory')
+def inventory_cli_arg(inventory_sources):
+    inventory_cli_arg, _ = inventory_sources
+    return inventory_cli_arg
+
+
+@pytest.fixture(scope="session")
+def inventory(inventory_sources):
+    inventory_cli_arg, inventory_env_var = inventory_sources
+    return inventory_cli_arg or inventory_env_var
+
+
+@pytest.fixture(scope="session")
+def kubeconfig(inventory):
```

Comment:
Can we also get this from the environment, `KUBECONFIG`, instead of assuming it's part of the inventory? Requires a bit more setup code before test invocation, but not that much.